doc/man/8/mount.ceph.rst

```diff
@@ -199,6 +199,51 @@ Advanced
     option is enabled, a namespace operation may complete before the MDS
     replies, if it has sufficient capabilities to do so.
 
+:command:`cephx_sign_messages`
+   Enable message signing for msgr1 on-the-wire protocol.  For msgr2.1
+   protocol this option is ignored: message signing is built into 'secure'
+   mode and not offered in 'crc' mode. This is the default.
+
+:command:`nocephx_sign_messages`
+   Disable message signing for msgr1 on-the-wire protocol. For msgr2.1
+   protocol this option is ignored.
+
+:command:`crush_location=x`
+    Specify the location of the client in terms of CRUSH hierarchy.  This
+    is a set of key-value pairs separated from each other by '|', with keys
+    separated from values by ':'.  Note that '|' may need to be quoted or
+    escaped to avoid it being interpreted as a pipe by the shell. The key
+    is the bucket type name (e.g. rack, datacenter or region with default
+    bucket types) and the value is the bucket name. For example, to indicate
+    that the client is local to rack "myrack", data center "mydc" and region
+    "myregion"::
+
+      crush_location=rack:myrack|datacenter:mydc|region:myregion
+
+    Each key-value pair stands on its own: "myrack" doesn't need to reside in
+    "mydc", which in turn doesn't need to reside in "myregion".  The location
+    is not a path to the root of the hierarchy but rather a set of nodes that
+    are matched independently, owning to the fact that bucket names are unique
+    within a CRUSH map.  "Multipath" locations are supported, so it is possible
+    to indicate locality for multiple parallel hierarchies::
+
+      crush_location=rack:myrack1|rack:myrack2|datacenter:mydc
+
+:command:`read_from_replica=<no|balance|localize>`
+    - ``no``: disable replica reads, always pick the primary OSD (Default).
+
+    - ``balance``: when issued a read on a replicated pool, pick a random
+      OSD for serving it. Otherwise it should be limited to read-only workloads
+      such as images mapped read-only everywhere or snapshots
```

Comment:
"... images mapped ..." is specific to RBD, need to adjust when cut-and-pasting ;)

Also, why sentence about the safety for general use only since Octopus was dropped?  This sentence (the one that starts with "Otherwise ...") is tied to it and doesn't make sense on its own.

Reply:
> "... images mapped ..." is specific to RBD, need to adjust when cut-and-pasting ;)
> 

Hmm, I just copied it and didn't notice this. Will fix it.

> Also, why sentence about the safety for general use only since Octopus was dropped? This sentence (the one that starts with "Otherwise ...") is tied to it and doesn't make sense on its own.

Just lost it by mistake and I will add it back.

Reply:
> Suggest
> 
> > * `balance`: When a replicated pool receives a read request, pick a random OSD from the PG's acting set to serve it.

Yeah, looks better. Thanks